packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+import { hashLabelValues } from './Utils';
+import { CounterHandle } from './Handle';
+
+export class Metric<T> implements types.Metric<T> {
+  private readonly _monotonic: boolean;
+  private readonly _disabled: boolean;
+  private readonly _handles: Map<String, T> = new Map();
+
+  constructor(name: string, options: types.MetricOptions) {
+    this._monotonic = options.monotonic!;
```

Comment:
Instead of doing a force unwrap (`!`) can we use some type of default value if this is `undefined` or `null`? Same for disabled.

If the default is false, can we just do `this._monotonic = !!options.monotonic;`?

Reply:
I have created [SDK specific interface](https://github.com/open-telemetry/opentelemetry-js/pull/415/files#diff-e3728f5692d938914d82d335cd641267R18) ([with default values](https://github.com/open-telemetry/opentelemetry-js/pull/415/files#diff-e3728f5692d938914d82d335cd641267R42)) to avoid doing a force unwrap(`!`).